packages/code-studio/src/styleguide/StyleGuideRoot.tsx

```diff
@@ -4,12 +4,17 @@ import { FontBootstrap } from '@deephaven/app-utils';
 import '@deephaven/components/scss/BaseStyleSheet.scss';
 import { MonacoUtils } from '@deephaven/console';
 import { store } from '@deephaven/redux';
+import { DownloadServiceWorkerUtils } from '@deephaven/iris-grid';
 import MonacoWorker from 'monaco-editor/esm/vs/editor/editor.worker?worker';
-import DownloadServiceWorkerUtils from '../DownloadServiceWorkerUtils';
 import StyleGuideInit from './StyleGuideInit';
 
 export function StyleGuideRoot(): JSX.Element {
-  DownloadServiceWorkerUtils.registerOnLoaded();
+  DownloadServiceWorkerUtils.register(
+    new URL(
+      `${import.meta.env.BASE_URL ?? ''}download/serviceWorker.js`,
```

Comment:
I don't think it conflicts. The first change cleaned up having like 3 spots where we concatenated the path w/ the string literal `download/`. This change removes the need for `IrisGridPanel` to receive a `getDownloadWorker` prop or receive that URL via redux/context from the app.

This actually moves 1 step closer to the download worker being part of `iris-grid` (all of the logic of the worker is for `iris-grid`). If we moved `serviceWorker.js` into `iris-grid` and exposed it somehow, we could do what monaco does basically, but a bit different. Service workers have to be served from a URL, so it would be up to the app to place the worker file into some location at build time and then register w/ wherever it was placed.